Orange/canvas/scheme/link.py

```diff
@@ -83,15 +83,17 @@ class State(enum.IntEnum):
         """
         Flags indicating the runtime state of a link
         """
+        #: The link has no associated state.
+        NoState = 0
         #: A link is empty when it has no value on it
-        Empty = 0
+        Empty = 1
         #: A link is active when the source node provides a value on output
-        Active = 1
+        Active = 2
         #: A link is pending when it's sink node has not yet been notified
         #: of a change (note that Empty|Pending is a valid state)
-        Pending = 2
+        Pending = 4
```

Comment:
Is Pending a valid state or is it always Empty|Pending or Active|Pending?

Reply:
Let's say it is valid.

Reply:
I asked because it used to be marked with a dashed line (it was not active), but it is marked with a solid line in this PR (it is not empty). If the state does not occur normally by itself, it is not a problem.